### ISSUE TITLE
[BUG] frigate sub labels aren't updated if stop_on_match is set to false

### ISSUE THREAD

**Issue by NickM-27** [jakowenko/double-take#268]
**Describe the bug**
When disabling stop_on_match, frigate sub labels are not sent.

**Version of Double Take**
`0.13.1`

**Expected behavior**
Frigate sub labels should be set even when not stopping on match

**Comment by skrashevich** [comment]
Double-take in this case concatenate all founded names into one line; and on the frigate side it does'nt pass length check of 20 characters:
```
 if new_sub_label and len(new_sub_label) > 20:
        return make_response(
            jsonify(
                {
                    "success": False,
                    "message": new_sub_label
                    + " exceeds the 20 character limit for sub_label",
                }
            ),
            400,
        )
```
this length limit is really necessary?

**Comment by NickM-27** [comment]
Yes it is, because the length of the db field is max 20 characters

**Comment by skrashevich** [comment]
> Yes it is, because the length of the db field is max 20 characters

maybe, truncate sub_label on frigate side instead of generating an error?
`new_sub_label = new_sub_label[:20] `
